app/views/documents/spouse_ssn_itins/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<% content_for :form_question, "Attach a photo of your spouse's Social Security Card or ITIN" %>
+
+<% content_for :form_help_text, "The IRS requires us to see a valid SSN or ITIN for tax preparation services." %>
```

Comment:
should we put "for your spouse" somewhere in here?

Reply:
could! the ticket didn't specify but that seems helpful